tmt/hardware.py

```diff
@@ -1080,6 +1080,30 @@ def _parse_disks(spec: Spec) -> BaseConstraint:
     return group
 
 
+@ungroupify
+def _parse_bootdisks(spec: Spec) -> BaseConstraint:
```

Comment:
It's parsing the `boot-disk` requirement, therefore `_parse_boot_disk` would be a better fit.